wagtail/admin/navigation.py

```diff
@@ -1,9 +1,19 @@
+import warnings
+
 from django.conf import settings
 
 from wagtail.permission_policies.pages import PagePermissionPolicy
+from wagtail.utils.deprecation import RemovedInWagtail60Warning
 
 
 def get_pages_with_direct_explore_permission(user):
```

Comment:
Other than tests, this function is never used directly in Wagtail. Previously, it was used in `get_explorable_root_page`. We can remove this if desired.